primitives/src/locktime/absolute.rs

```diff
@@ -438,10 +451,15 @@ mod tests {
     }
 
     #[test]
-    fn lock_time_from_invalid_hex_should_err() {
-        let hex = "0xzb93";
-        let result = LockTime::from_hex(hex);
-        assert!(result.is_err());
+    fn invalid_hex() {
+        assert!(LockTime::from_hex("0xzb93").is_err());
+        assert!(LockTime::from_unprefixed_hex("zb93").is_err());
+    }
+
+    #[test]
+    fn invalid_locktime_type() {
+        assert!(LockTime::from_height(500_000_000).is_err()); // Above threshold.
+        assert!(LockTime::from_time(499_999_999).is_err()); // Below threshold.
```

Comment:
If the aim is brain-dead easy to read tests perhaps this would work?

```rust
    fn invalid_locktime_type() {
        assert!(LockTime::from_height(499_999_999).is_ok()); // Below the threshold.
        assert!(LockTime::from_height(500_000_000).is_err()); // The threshold.
        assert!(LockTime::from_height(500_000_001).is_err()); // Above the threshold.

        assert!(LockTime::from_time(499_999_999).is_err()); // Below the threshold.
        assert!(LockTime::from_time(500_000_000).is_ok()); // The threshold.
        assert!(LockTime::from_time(500_000_001).is_ok()); // Above the threshold.
    }
```

Reply:
Thanks for the suggestions.  I agree they are easier to read and will make the changes if I need to make any other edits before this is merged.

Reply:
Suggested change made